DNN Platform/Library/Services/GeneratedImage/Constants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DotNetNuke.Services.GeneratedImage
+{
+    class Constants
+    {
+        internal const string UserIdListToClearDiskImageCacheKey = "UserIdListToClearDiskImage_{0}";
+    }
+}
```

Comment:
No need for a new file for this; just add this to the ``public class DataCache`` in the ``DataCache.cs`` file same as all other cache keys with all timeouts.